### DISCUSSION TITLE
Regarding verification whether new architecture is enabled or not?

### DISCUSSION THREAD

**Discussion by TheTushar2696** [reactwg/react-native-new-architecture#119] [Q&A] [answered]
Is it possible that I dont see concurrentRoot flag in terminal even though new architecture is enabled?
Scenario:
I've enabled isNewArchitectureEnabled flag in gradle.properties and I can se all the logs in the build related to Cmake and all but I am unable to see concurrentRoot true is terminal, I am just able to see below
<img width="467" alt="Screenshot 1944-12-19 at 11 07 34 AM" src="https://user-images.githubusercontent.com/40525668/224233725-c85ae239-2b72-443a-a832-8a909ac4fa88.png">

**Comment by cortinico** [comment, marked as answer]
> Is it possible that I dont see concurrentRoot flag in terminal even though new architecture is enabled?

This looks like a bug. You should see a note that Fabric and Concurrent Root are enabled on Console. Those are the parameters that are passed over to the application once running, and are printed on console.
If you don't see them, your setup is most likely wrong.
